### ISSUE TITLE
Linking instagram.

### ISSUE THREAD

**Issue by AlexK1a2d** [tomlandia/fuckr#159]
Is there a way to link to your instagram profile like you would on regular Grindr to verify yourself?

**Comment by youngbond** [comment]
Yes, but you will have to log out and log in on the mobile app. You can create an account on the mobile app to fill out all profile details, close or delete the app, and then log in on fuckr.

**Comment by Grindr13** [comment]
You won't be able to use Fuckr anymore now #157 

> Closes #159
